### DISCUSSION TITLE
How to add words to user's dictionary

### DISCUSSION THREAD

**Discussion by ryanxingql** [znck/grammarly#294] [Q&A] [answered]
I want to add some words to my dictionary, so that Grammarly would not raise a warning about the spelling of this word.
Thank you.

**Comment by lara-rium** [comment]
I'm wondering the same thing...

**Reply by trebeljahr** [reply]
same here.

**Comment by znck** [comment, marked as answer]
User dictionary is read-only. You can add words to your dictionary using https://account.grammarly.com/customize.

**Reply by OnkelTem** [reply]
Wow, it's so inconvenient, tbh.
I wish there was a way to add a word by RMB.

**Reply by lara-rium** [reply]
same, i'd expect that to be a feature by default

**Reply by rshade** [reply]
There should be a .grammarly that can ignore these locally.

**Reply by Bruncky** [reply]
I tried doing that, but it seems to take a very long time for the extension to update. Is there a way to force a sync?

**UPDATE (24.10):** I'm still waiting for custom words to show up in VS Code, it's been about 5 days total now. Am I missing something?

**Reply by ssbarnea** [reply]
User dictionary does not make sense for open-source projects. We would need an in-repo dictionary, one where we can configure the path to the file. That is essential as many projects do you spellcheckers and they already have their own dictionaries -- nobody wants to maintain multiple dictionaries.
